src/charm.py

```diff
@@ -401,6 +401,7 @@ def _on_config_changed(self, event: ConfigChangedEvent) -> None:
             return
         if vault.is_sealed():
             vault.unseal(unseal_keys=unseal_keys)
+        vault.enable_audit_device(device_type="file", path="stdout")
```

Comment:
Looking at the integration tests run, it looks like we run this too quickly after the unseal. there may be a validation to add here.

Reply:
✅